getting-started/compile-contract.md

```diff
@@ -0,0 +1,63 @@
+---
+order: 4
+---
+
+# Download and Compile a Contract
+
+In this section, we will download a sample contract, compile to it to wasm binary executable.
+
+Please first review the [client setup instructions](./setting-env.md), and configure and verify a client, either Go CLI or
+Node.JS console.
+
+## Compiling and Testing a Contract
+
+Let's download the repo which we collect [`cosmwasm-examples`](https://github.com/CosmWasm/cosmwasm-examples) and try out an existing simple escrow contract that can hold some native tokens and gives the power to an arbiter to release them to a pre-defined beneficiary. First clone the repo and try to build the wasm bundle:
+
+```bash
+# get the code
+git clone https://github.com/CosmWasm/cosmwasm-examples
+cd cosmwasm-examples/escrow
+git checkout escrow-0.5.2
```

Comment:
We need `escrow-0.6.0` to be compatible with `wasmd v0.10.0`. This label did not exist until just now (I forgot when making the 0.10 update) and I just pushed it.